cmd/api-headers.go

```diff
@@ -95,6 +96,13 @@ func setObjectHeaders(w http.ResponseWriter, objInfo ObjectInfo, rs *HTTPRangeSp
 		w.Header().Set(xhttp.XCache, objInfo.CacheStatus.String())
 		w.Header().Set(xhttp.XCacheLookup, objInfo.CacheLookupStatus.String())
 	}
+
+	// Set tag count if object has tags
+	tagCount := len(strings.Split(objInfo.UserTags, "&"))
```

Comment:
This may not be a valid way to calculate this, you should parse `objInfo.UserTags` with url.ParseQuery() and then calculate the `len(url.Values)`